traffic_ops/traffic_ops_golang/deliveryservicesregexes/deliveryservicesregexes.go

```diff
@@ -227,13 +229,21 @@ func Post(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	dsID := inf.IntParams["dsid"]
+	dsName, _, err := dbhelpers.GetDSNameFromID(inf.Tx.Tx, dsID)
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("getting delivery service name from id: "+err.Error()))
+		return
+	}
+
 	respObj := tc.DeliveryServiceIDRegex{
 		ID:        regexID,
 		Pattern:   dsr.Pattern,
 		Type:      dsr.Type,
 		TypeName:  typeName,
 		SetNumber: dsr.SetNumber,
 	}
+	api.CreateChangeLogRawTx(api.ApiChange, "DS: "+string(dsName)+", ID: "+strconv.Itoa(dsID)+", ACTION: Created a "+strconv.Itoa(dsr.Type)+" regular expression ("+dsr.Pattern+") in position "+strconv.Itoa(dsr.SetNumber), inf.User, inf.Tx.Tx)
```

Comment:
this results in

`DS: foo, ID: 10773, ACTION: Created a 18 regular expression (boo) in position 1`

just get rid of the "+strconv.Itoa(dsr.Type)+"